web/packages/teleport/src/features.tsx

```diff
@@ -58,8 +58,7 @@ const Audit = lazy(() => import('./Audit'));
 const Nodes = lazy(() => import('./Nodes'));
 const Sessions = lazy(() => import('./Sessions'));
 const UnifiedResources = lazy(() => import('./UnifiedResources'));
-const Account = lazy(() => import('./Account'));
-const AccountNew = lazy(() => import('./Account/AccountNew'));
+const Account = lazy(() => import('./Account/AccountNew'));
```

Comment:
Will we eventually rename `AccountNew` to `Account` and get rid of the old one?

I don't like the idea of having `New` or `Next` on things. In a few years we'll end up with `AccountNewNew` etc.

Reply:
Yes, we will. I save it for the next week, though, as right now I have to prioritize squeezing in more impactful changes.

Reply:
`AccessRequestRequest`